code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -555,8 +555,8 @@
 ///Displays a special speech bubble when someone inside the mecha speaks
 /obj/vehicle/sealed/mecha/proc/display_speech_bubble(datum/source, list/speech_args)
 	SIGNAL_HANDLER
-	var/list/speech_bubble_recipients = get_hearers_in_view(7,src)
-	for(var/mob/M in speech_bubble_recipients)
+	var/list/speech_bubble_recipients = list()
+	for(var/mob/M in get_hearers_in_view(7,src))
 		if(M.client)
```

Comment:
???

Reply:
This was causing runtimes everytime someone spoke in/near occupied mechs, as get_hearers_in_view was adding mechs or some other stuff to speech_bubble_recipients, instead of just the occupants.